# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: intl/l10n/FluentBundle.h
# Commit: 3404fd6725f1
# Full Hash: 3404fd6725f15d1253a7f76e3937185bd353f263
# Author: Zibi Braniecki <zbraniecki@mozilla.com>
# Date: 2021-08-02 21:43:56
# Regressor Bug: 1613705
# File Overlap Count: 1
# Description:
#   Bug 1613705 - [localization] part2: Switch Localization class to use localization-ffi. r=emilio,nika
#   
#   Depends on D104788
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D104789
# ==============================================================================

diff -r c9cfbba15d90 -r 3404fd6725f1 intl/l10n/FluentBundle.h
--- a/intl/l10n/FluentBundle.h	Mon Aug 02 09:54:07 2021 +0000
+++ b/intl/l10n/FluentBundle.h	Mon Aug 02 09:54:07 2021 +0000
@@ -81,6 +81,9 @@
                      const dom::Optional<JS::Handle<JSObject*>>& aErrors,
                      nsACString& aRetVal, ErrorResult& aRv);
 
+  static void ConvertArgs(const L10nArgs& aArgs,
+                          nsTArray<ffi::L10nArg>& aRetVal);
+
  protected:
   virtual ~FluentBundle();
 